### ISSUE TITLE
Flashing notification for channel

### ISSUE THREAD

**Issue by disconnect00** [EionRobb/purple-discord#303]
Hi,
I have encountered an issue that channel in taskbar does not flash orange when channel receives new message. Is there a way to fix it?

Thank you :)

**Comment by EionRobb** [comment]
Normally that would only happen if someone mentions your name, but you should be able to use the "Message Notifications" plugin in Tools->Plugins
![image](https://user-images.githubusercontent.com/1063865/88770998-b3e9c200-d1d2-11ea-99ef-a9fd9701084c.png)
if you tick the "Chat Messages" one then it should notify you when all chat messages come through

Unfortunately there's not too much we can do from a protocol plugin to affect the  UI like that

**Comment by disconnect00** [comment]
Hi Robb, thank you for this wonderful plugin.
I would like to provide more information regarding this issue:
- My direct message turns orange on taskbar normally when I receive new message.
- I encounter this issue after I updated Discord plugin to fix crashing issue few months ago (I remember that the changelog was something like "Fixing 2=3". Before updating, channels showed up and there was flashing orange bar on taskbar. After updating, channels dont pop up anymore when there are new messages (number of people is below 80).
- Sound notification works fine. I still hear the ping sound when there are new messages.

I dont know which settings I turned off, or is it because of the update. The left one in screenshot is direct message and it works fine (I clicked on it so it's black now), the right one is my channel.
![image](https://user-images.githubusercontent.com/68916945/88897650-daad0480-d275-11ea-8d69-dcd6d8e66143.png)
